sections/csgcn/csgcn-chapter.tex

```diff
@@ -218,6 +218,7 @@ \subsection{Adjacency matrix}\label{subsec:csgcn_is_adj_mat}
 Since there are no interactions between users and users, the first quadrant is a zero-matrix of size $$|U|\times|U|$$ where $|U|$ is the amount of users in the dataset.
 This goes for all the zero matrices seen on $A$, making it mostly sparse.
 Adding the side-information to the adjacency matrix allows us to express the connections between entities (users and items) and their side-information in the graph, allowing it to be used for higher-order connectivity in the convolution layers.
+It is worth noting that unlike traditional GCNs, CSGCN does not utilize self-connections for, since the layer combination performed after the convolution layers captures the same effect as self-connections through the 0th layer.
```

Comment:
```suggestion
It is worth noting that unlike traditional GCNs, CSGCN does not utilize self-connections since the layer combination performed after the convolution layers captures the same effect as self-connections through the 0th layer.
```